pages/design-tokens/typesetting/overview.md

```diff
@@ -477,7 +477,9 @@ The individual design token sections go into this in more detail, but here's a s
 </div>
 
 ### Family, size, and line height together
-The `typeset()` mixin outlined below accepts the tokens listed as well as a special `null` value. If either of the three properties get `null` instead of a token, the system will use a default value from `$uswds-theme-typography`: either `$theme-body-font-family`, `$theme-body-font-size`, or `$theme-body-line-height` depending on the property passed the `null` value.
+The `typeset()` mixin accepts the tokens listed as well as a special `null` value. If either of the three properties get `null` instead of a token, the system will use a default value from [typography settings]({{ site.baseurl }}/documentation/settings/#typography-settings): either `$theme-body-font-family`, `$theme-body-font-size`, or `$theme-body-line-height` depending on the property passed the `null` value.
+
+See this example:
```

Comment:
What example?

Reply:
The section beneath shows examples of the `typeset()` mixin with various settings and then the large code block shows what that looks like in practice. This was a change requested by @bonnieAcameron 

https://github.com/uswds/uswds-site/pull/1866#discussion_r1010888875


Would we want to change the wording so that it references multiple examples? Something like "_For example_" or "_See these examples_" ?